pandas/tests/io/json/test_pandas.py

```diff
@@ -813,6 +813,41 @@ def test_reconstruction_index(self):
         result = read_json(df.to_json())
         assert_frame_equal(result, df)
 
+    @pytest.mark.parametrize(
+        "date_format,delta,formatted",
+        [
+            ("iso", ["1D", "2D"], ["P1DT0H0M0S", "P2DT0H0M0S"]),
+            ("epoch", ["1D", "2D"], [86400000, 172800000]),
+        ],
+    )
+    def test_series_timedelta_to_json(self, date_format, delta, formatted):
+        # GH28156: to_json not correctly formatting Timedelta
+        s = Series([pd.Timedelta(d) for d in delta])
+        s.index = ["one", "two"]
```

Comment:
Is modifying the index critical for the test? If not best to exclude

Reply:
I wouldn't say it's critical.  I had initially thought to include it because in a previous iteration I had some code in there that was resetting the index and therefore causing a test like this to fail.  But I can see that if the changes are sufficiently well-constrained to not mess with anything outside of what it should do, then changing the index in the test might not be critical.